src/components/DropdownMenu.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="dropdowns">
+    <div class="dropdown">
+      <button @click="openSort">
+        <div class="dropdown-trigger">
+          <p>並び替え</p>
+          <span class="ti-angle-down"></span>
+        </div>
+      </button>
+      <div v-if="state.isOpenSort" class="dropdown-menu">
```

Comment:
```suggestion
      <div v-if="state.isOpenSort" :class="$style.dropdown-menu">
```
っていう風にして`style`タグの中を直してほしい(他の`class`を使っているところも全部)(`style`タグの中については以下のレビューを参照)
参考: https://jp.vuejs.org/v2/guide/class-and-style.html